packages/ui/components/ui/tx/view/transaction.tsx

```diff
@@ -12,7 +12,9 @@ export const TransactionViewComponent = ({ txv }: { txv: TransactionView }) => {
 
   return (
     <div className='flex flex-col gap-8'>
-      <MemoViewComponent memo={txv.bodyView.memoView} />
+      {txv.bodyView.memoView?.memoView.case !== undefined && (
```

Comment:
@turbocrime I had to change back to `memoView.case` instead of `memoView` here, since `new MemoView` automatically inserts `{ case: undefined }` (I guess it is a gRPC codegen thing)

Reply:
actually just kidding, I was able to fix this by re-arranging the conditional in `perspective.ts`